Apps/PatientDataAccess/src/HealthData.cs

```diff
@@ -0,0 +1,108 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.PatientDataAccess
+{
+#pragma warning disable SA1201 // Elements should appear in the correct order
+    using System;
+
+    /// <summary>
+    /// abstract record for health data.
+    /// </summary>
+    public abstract record HealthData;
+
+    /// <summary>
+    /// The details of a diagnostic imaging exam.
+    /// </summary>
+    public record DiagnosticImagingExam : HealthData
```

Comment:
Similar to the previous comment about the extra enum, this looks like the 3rd model for this data.  What different purpose does this DiagnosticImagingExam that inherits from HealthData serve compared to the one that inherits from HealthDataEntry?